### DISCUSSION TITLE
各位绳匠中秋节快乐

### DISCUSSION THREAD

**Discussion by cyq20091029** [share121/inter-knot#5020] [灌水]
<img width="720" alt="截屏2024-09-17 03 10 36" src="https://github.com/user-attachments/assets/de060016-36c3-48fe-a7f9-67a435dd7ef0">
万恶的学校要我们在中秋节晚上上晚自习，不知道想出这种政策的校长是不是没有家人来团聚

**Comment by swyh01000** [comment]
我们还挺人性化，周三早上再去

**Reply by cyq20091029** [reply]
羡慕了<img src="https://img-static.mihoyo.com/communityweb/upload/a7304904a31329464114b1074218967e.png" alt="帕姆 瘫倒" width="75" >

**Comment by passbyDFX** [comment]
中秋快乐

**Comment by KELERSAMA** [comment]
星期日补过课了 所以星期二不用去
<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >

**Comment by zdsy03** [comment]
能放学回家和家人在一起已经很好啦，中秋快乐🎉！

**Comment by Kazuha-hhh** [comment]
我们这晚上刮风打雷下大雨，月亮看不到
